### ISSUE TITLE
torch.nn.instancenorm, why running_mean/var's shape is C, not N * C

### ISSUE THREAD

**Issue by wei61547-jp** [pytorch/pytorch#59864]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by wei61547-jp** [comment]
why running_mean/var's shape is C, not N * C
according to the https://arxiv.org/abs/1607.08022, the shape of mean/var is N *C , but for torch.nn.instancenorm ,
    at::alias(running_mean).copy_(running_mean_.view({ b, c }).mean(0, false));  }
i don't know the operation of "mean(0,false)" is use for ?
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Normalization.cpp#L581

**Comment by soulitzer** [comment]
Issues are for bugs and features only, please ask your question on the forum https://discuss.pytorch.org/.
